### PR TITLE
throw error in wallet-adapter if sdk creds not supplied

### PR DESCRIPTION
**Pull request by gklawcode**

### CODE REVIEW

**Review by tsmbl — CHANGES_REQUESTED** [review]
Awesome proposal, I think we should also fix tests before merging

**Comment by zhelezkov** [comment]
any upd?

**Comment by gklawcode** [comment]
agreed, will fix tests once finished with new wallet route work today on other PR

**Comment by gklawcode** [comment]
tests passing @tsmbl @zhelezkov

**Review by tsmbl — DISMISSED** [review]
Awesome, several small comments
1. we need also to update examples in code and gitbook
2. update changelog

File: packages/blockchain-sdk-aptos/src/wallet-adapter/node-dialect-aptos-wallet-adapter.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export class NodeDialectAptosWalletAdapter
       );
       return new NodeDialectAptosWalletAdapter(account);
     }
-    if (process.env.DIALECT_SDK_CREDENTIALS) {
+    else if (process.env.DIALECT_SDK_CREDENTIALS) {
```
Comment (zhelezkov) [inline packages/blockchain-sdk-aptos/src/wallet-adapter/node-dialect-aptos-wallet-adapter.ts:33]: is this code formatting issue?

**Review by zhelezkov — APPROVED** [review]
let's fix code formatting
if all tests are green and no compilation then it's gtg

File: packages/blockchain-sdk-aptos/src/wallet-adapter/node-dialect-aptos-wallet-adapter.ts (hunk 1 above)
Comment (gklawcode) [reply packages/blockchain-sdk-aptos/src/wallet-adapter/node-dialect-aptos-wallet-adapter.ts:33]: no this was an intended change

Comment (gklawcode) [reply packages/blockchain-sdk-aptos/src/wallet-adapter/node-dialect-aptos-wallet-adapter.ts:33]: it is now if, elif, else pattern -- prev just if, if return early

Comment (gklawcode) [reply packages/blockchain-sdk-aptos/src/wallet-adapter/node-dialect-aptos-wallet-adapter.ts:33]: @zhelezkov

Comment (gklawcode) [reply packages/blockchain-sdk-aptos/src/wallet-adapter/node-dialect-aptos-wallet-adapter.ts]: have run prettier, thx!

**Pull request merged**